docs/cloud-providers/azure/azure-provider.md

```diff
@@ -41,26 +41,17 @@ You might see an error similar to the following, but that is OK, the permissions
 Operation failed with status: 'Conflict'. Details: 409 Client Error: Conflict for url: https://graph.windows.net/e7985bc4-a3b3-4f37-b9d2-fa256023b1ae/oauth2PermissionGrants?api-version=1.6
 ```
 
-After these steps are completed, you should have the following file on your local filesystem:
-
-* `crossplane-azure-provider-key.json`
-
-## Grant Consent to Application Permissions
+Finally, you need to grant admin permissions on the Azure Active Directory to the service principal because it will need to create other service principals for your `AKSCluster`:
+```bash
+# gra
+az ad app permission admin-consent --id "${AZURE_CLIENT_ID}"
```

Comment:
this single step using `az` appears to be much less complicated than the UI sequence, which is awesome.

In your testing, did you do this with a new account that didn't already have these perms enabled? i'm hoping we have confidence that this step is reliable for everyone, as opposed to being vetted only by the account we normally use for testing that already had this set.

Reply:
By account, do you mean user account or service principal that is created in the first step?

I think the account I used is an old one with `Global Administrator` role but I created a new service principal in the first step and granted admin-consent to that new service principal with this command. As long as the account that user logged in for `az` command has the `Global Administrator` role, which is mentioned as a note, I think it shouldn't matter. Do you have a different experience?

Reply:
I think you're right about that distinction between user account and service principal, i think your instructions are the streamlined CLI equivalent to what we had previously for GUI steps.  Great!! 🎉